Frameworks/UIKit/WYPopoverController.mm

```diff
@@ -2634,8 +2639,12 @@ - (void)dismissPopoverAnimated:(BOOL)aAnimated
         }
 #endif
     };
+
+    UIPopoverPresentationController* popoverPresentationController = [viewController popoverPresentationController];
     
     void (^completionBlock)() = ^() {
+        // Keep popoverPresentationController alive via block-capture during teardown.
+        (void)popoverPresentationController;
```

Comment:
> (void)popoverPresentationController; [](start = 7, length = 37)

nit: might be a bit more clear to do an assignment to a local variable 'strongPopoverPresentationController', rather than this.